vllm/v1/core/kv_cache_manager.py

```diff
@@ -299,26 +272,12 @@ def reset_prefix_cache(self) -> bool:
             bool: True if the prefix cache is successfully reset,
             False otherwise.
         """
-        num_used_blocks = (self.num_gpu_blocks -
-                           self.free_block_queue.num_free_blocks)
-        if num_used_blocks > 0:
-            logger.warning(
-                "Failed to reset prefix cache because some "
-                "blocks (%d) are not freed yet", num_used_blocks)
+        if self.block_pool.reset_prefix_cache():
+            self.prefix_cache_stats.reset = True
+            return True
+        else:
             return False
```

Comment:
No need to put "return False" in the else-branch when the if-branch has a return

Reply:
addressed in https://github.com/vllm-project/vllm/pull/13973/commits/676f6621f9f88d5689635d2a502818ccadff40b7